content/release-notes/1.12.0.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2020-01-31T19:55:39-08:00
+linktitle: "1.12.0"
+title: "1.12.0"
+kubernetes: "1.15, 1.16, and 1.17"
+---
+{{<features>}}
+* Namespace can now be specified for [Helm charts in KOTS](https://kots.io/reference/template-functions/static-context/)
```

Comment:
I think this link is wrong, it should link to https://kots.io/reference/v1beta1/helmchart/#namespace